pandas/_libs/tslibs/timedeltas.pxd

```diff
@@ -3,7 +3,7 @@
 from numpy cimport int64_t
 
 # Exposed for tslib, not intended for outside use.
-cdef parse_timedelta_string(object ts)
+cpdef parse_timedelta_string(object ts, object specified_unit=*)
```

Comment:
If this isn't being cimported by any other cython modules, then remove it from the pxd entirely.

Reply:
Ok, other functions also use this so I will leave it.